src/examples/compliance.ts

```diff
@@ -0,0 +1,178 @@
+import { BigNumber } from '@polymeshassociation/polymesh-sdk';
+import {
+  ClaimType,
+  ConditionTarget,
+  ConditionType,
+  ScopeType,
+} from '@polymeshassociation/polymesh-sdk/types';
+
+import { getClient } from '~/common/client';
+import { toHumanObject } from '~/common/utils';
+
+/*
+  This script showcases Compliance related functionality. Covered functionality:
+    - Setting Compliance rules 
+    - Getting existing Compliance rules
+    - Pausing Compliance rules
+    - Unpausing Compliance rules
+    - Adding new requirements to existing Compliance rules
+    - Removing a Compliance rule
+    - Setting trusted Claim issuers
+    - Getting existing trusted Claim issuers
+    - Adding a new Claim issuer
+    - Removing a Claim issuer
+*/
+(async (): Promise<void> => {
+  console.log('Connecting to the node...\n\n');
+  const api = await getClient(process.env.ACCOUNT_SEED);
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
What about:

```js
if(!id) throw new Error('Signing Identity was found')
```

instead of non-null assertion?

Reply:
This pattern was followed in other examples as well so I carried that. I can change in all if you would like